### ISSUE TITLE
Re-Enable Location doesn't Re-Enable the Location-Settings

### ISSUE THREAD

**Issue by fblehnert** [ChrisTitusTech/winutil#1690]
**Describe the bug**
After Debloating a new Win11-Installation by using the "recommended selection" for Laptop I cant Enable Location to use "find my device" from the MS Dashboard due to the fact that the Location was disabled from the tool which is fine so far.

But if i now enable the location again by using the button undo selected tweaks I still cant enable the location in the settings since Windows is still telling me that this settings are disabled from the adminstrator.

**To Reproduce**
1. Go to 'Tweaks'
2. Click on 'Laptop' in 'Recommended Selection'
3. Click on 'Run Tweaks'
4. Reboot
5. Result: You should now see a banner under Windows Location-Settings which tells you: These settings are managed by your Organisation
6. Restart Utility
7. Go to 'Tweaks'
8. Click on 'Laptop' in 'Recommended Selection'
9. Click on 'Undo Selected Tweaks'
10. Reboot
11. Result: You should now still see banner under Windows Location-Settings which tells you: These settings are managed by your Organisation

**Expected behavior**
After undoing the tweaks and rebooting the device I would expect that I am now able to Manage the Location-Settings again by myself.

**Screenshots**
Sorry I got a German Version of Win11 so the Screenshot is also German. But I hope you know what I mean.

![image](https://github.com/ChrisTitusTech/winutil/assets/80295330/cb65c19d-21f6-4ecc-8246-39dd9de12062)

**Comment by Marterich** [comment]
Hi, I've looked at the problem you described, and most likely found the root of the problem. 

TLDR: (Workaround) 
Open the Terminal (or Powershell) as Admin and Run 
`Remove-Item -Path "HKLM:\SOFTWARE\Policies\Microsoft\Windows\LocationAndSensors"`
This will remove the location policy applied by OO Shutup

First of all a bit of context.
The Tweak "Run OO Shutup" runs a bunch of privacy-focused tweaks according to the config file [ooshutup10_winutil_settings.cfg](https://github.com/ChrisTitusTech/winutil/blob/main/ooshutup10_winutil_settings.cfg). OO Shutup works by modifying the registry keys that would also be used via Group Policies in a corporate setting to specify policies for the users and/or computers in your company that you want to be enforced.

In the mentioned config file, the following options regarding policies for the use of location data are enabled.

- P056: Disables app access to device location [...]
- C007: Cortana and search are disallowed to use location [...]
- L001: Disable functionality to locate the system [...]
- L003: Disable scripting functionality to locate the system [...]

 To enforce the Policy the OO Shutup creates the Following Registry Key (if it doesn't exist)
`HKLM\Software\Policies\Microsoft\Windows\LocationAndSensors`
and adds the following DWORD values.

- Name: DisableLocation, Value: 1, Type: DWORD
- Name: AllowSearchToUseLocation, Value: 1, Type: DWORD
- Name: DisableWindowsLocationProvider, Value: 1, Type: DWORD
- Name: DisableLocationScripting, Value: 1, Type: DWORD

Proposals: 

1. In Winutil no undo action is set for OO Shutup, so even if you were to select "Run OO Shutup" -> Undo Selected Tweaks nothing would change. A solution could be to create a config file eg. ooshutup10_winutil_defaults.cfg which contains the "Factory Settings" which could revert all changes of OO when pressing undo. 
2. The undo action of the "Disable Location Tracking" could also be potentially modified to take the changes of OO Shutup into account and remove the Registry keys as well. The downside would be, that this approach would start mixing the actions of different tweaks so it becomes intransparent to the user what action does what exactly.
3. A point could also be made to completely remove the location settings from the OO Shutup config because for the end user it could seem that no option would modify location settings other than the Checkbox labeled "Disable Location Tracking". The Registry changes from OO Shutup could also be merged with the functionality of the "Disable Location Tracking" tweak.

I'm hesitant to select any of the solutions as "The One and Only" right one so if anyone would like to pitch in any input or preference is appreciated.
